db_graph.php

```diff
@@ -32,7 +32,7 @@
           <div class="input-group-addon">
             <i class="far fa-clock"></i>
           </div>
-          <input type="text" class="form-control pull-right" id="querytime" value="Click to select date and time range">
+          <input type="button" class="form-control pull-right" id="querytime" value="Click to select date and time range" style="text-align:left">
```

Comment:
This is how you'd do with a button.
```suggestion
          <button class="form-control pull-right" id="querytime">Click to select date and time range</button>
```

Reply:
> This is how you'd do with a button.

Testing the proposed change, ranges can be selected correctly.

However, with the `<input>` tag, the text would change to reflect the selected time range. If I clicked "Yesterday" I would see "December 19th 2019, 00:00 to December 19th 2019, 23:59" in the button, even with the type set to `button`. However, after changing it to an actual button, the text no longer changes and is always "Click to select date and time range" making it more difficult to see exactly what timeframe you are looking at. Something else may need to be changed as well to retain this functionality.

Reply:
Ah, thanks for checking. I had a quick look into the `daterangepicker` documentation and it apparently can only be set up on top of an `<input>` field. I'll be happy to approve your change when you remove the text align.